cost/terminate_policy/README.md

```diff
@@ -11,10 +11,17 @@ This policy relies on a tag to terminate instances based on a schedule.  The tag
 You can follow the following example when tagging your instances. If you tag in the cloud natively you can ignore `ec2`,`gce`,`azure`, Flexera CMP will automatically add that on discovery.
```

Comment:
This sentence reads funny.  replace follow with use.
*Yuo can use the following example ...*